app/views/public/loans/index.html.slim

```diff
@@ -13,8 +13,8 @@
           li class=('active' if @params[:status] == 'all')
             = link_to t(:show_all), @params.merge(status: 'all', pg: nil)
       .country
-        = label_tag(:country, "#{t :country}: ")
-        = select_tag(:country, options_for_select([[t(:show_all), 'all']] + @countries, @params[:country]),
+        = label_tag(:division, "#{t :division}: ")
+        = select_tag(:division, options_for_select([[t(:show_all_divisions), 'all divisions']] + @divisions, @params[:division]),
```

Comment:
Why not just use "all" for the value? I have a feeling the space character in that value will cause problems later. For that matter, the label could also be simply "All", since "Division" is displayed right next to it.

Reply:
👍